mono/mini/debugger-agent.c

```diff
@@ -115,10 +115,10 @@ typedef struct {
 
 typedef struct
 {
+	//Must be the first field to ensure pointer equivalence
+	DbgEngineStackFrame de;
 	int id;
-	guint32 il_offset, native_offset;
-	MonoDomain *domain;
-	MonoMethod *method;
+	guint32 il_offset;
```

Comment:
Wouldn't be easier to move the whole struct ?

Reply:
The problem I found with moving the whole struct is figuring out what fields I need to handle/init on the wasm side.